container/attribute.go

```diff
@@ -102,7 +102,7 @@ func setAttribute(c *Container, key, value string) {
 // Handler must not be nil.
 func iterateAttributes(c *Container, f func(*Attribute) bool) {
 	for _, a := range c.Attributes() {
-		if f(a) {
```

Comment:
We should use `f(&attrs[i])` instead, because the function may save the value in some place.